partner_event/wizard/res_partner_register_event_view.xml

```diff
@@ -1,25 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data>
+<!-- © 2016 Tecnativa S.L. - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
 
 <record id="res_partner_register_event_view" model="ir.ui.view">
     <field name="name">Register in an event</field>
     <field name="model">res.partner.register.event</field>
     <field name="arch" type="xml">
         <form string="Select event to register">
-            <group>
+            <group attrs="{'invisible': [('errors', '!=', False)]}">
                 <field name="event"/>
             </group>
+            <group attrs="{'invisible': [('errors', '=', False)]}">
+                <label nolabel="1" colspan="4"
+                       string="These partners haven't been registered because they're already registered or other error occurred"/>
+                <field nolabel="1" name="errors" colspan="4"/>
+            </group>
             <footer>
-                <button name="button_register"
-                        type="object"
-                        string="Create registrations"
-                        class="oe_highlight"
-                        />
-                or
-                <button special="cancel"
-                        string="Cancel"
-                        />
+                <div attrs="{'invisible': [('errors', '!=', False)]}">
+                    <button name="button_register"
+                            type="object"
+                            string="Create registrations"
+                            class="oe_highlight"
+                            />
+                    or
+                    <button special="cancel"
+                            string="Cancel"
+                            />
+                </div>
+                <div attrs="{'invisible': [('errors', '=', False)]}">
+                    <button special="cancel"
+                            string="OK"
```

Comment:
I think you can drop this. Users already have the X in the dialog, so not really adding anything interesting IMHO.